recipes/octomap/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "octomap" %}
+{% set version = "1.9.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/OctoMap/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 5f81c9a8cbc9526b2e725251cd3a829e5222a28201b394314002146d8b9214dd
+  patches:
+    # Compatibility with C++17
+    - byte.patch
+    - fix-exported-targets.patch  # [win]
+
+build:
+  number: 0
+  skip: true  # [win and vc<14]
+  run_exports:
+    - {{ pin_subpackage(name, max_pin='x.x') }}
+
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - make
+    - pkg-config
+
+test:
+  requires:
+    - pkg-config
+  commands:
+    - pkg-config --modversion {{ name }}
+    - pkg-config --libs {{ name }}
+    - test -f $PREFIX/lib/lib{{ name }}${SHLIB_EXT}  # [unix]
+    - if exist %LIBRARY_PREFIX%\lib\octomap.lib (exit 0) else (exit 1)  # [win]
+
+about:
+  home: https://github.com/OctoMap/octomap
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: octomap/LICENSE.txt
+  summary: 'An Efficient Probabilistic 3D Mapping Framework Based on Octrees'
+  dev_url: https://github.com/OctoMap/octomap
+
+extra:
+  recipe-maintainers:
+    - jcarpent
```

Comment:
@jcarpent, can you confirm if you're happy be listed as a maintainer?

Reply:
Yes, sure. I definitely confirm.